### PR TITLE
remove alt apostrophes

### PR DESCRIPTION
**Pull request by mongodben**
### Jira

n/a
### Staged Changes (Requires MongoDB Corp SSO)

n/a
### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/swift/examples/read-and-write-data.txt [hunk 1]
```diff
@@ -322,7 +322,7 @@ query.
 .. example::
 
    To get a result set for tan dogs, and tan dogs whose names start with
-   ‘B’, chain two queries like this:
+   ‘B', chain two queries like this:
```
Comment (dacharyc) [inline source/sdk/swift/examples/read-and-write-data.txt:325]:
```suggestion
   'B', chain two queries like this:
```
Weird that find and replace caught one of those but not the other!

**Review by dacharyc — APPROVED** [review]
Awesome! Just one spot where find-and-replace missed one but I'm so happy you did this ✋

File: source/sdk/swift/examples/read-and-write-data.txt (hunk 1 above)
Comment (mongodben) [reply source/sdk/swift/examples/read-and-write-data.txt:325]: nice catch! b/c this is a different character! `‘` vs `’`

Comment (mongodben) [reply source/sdk/swift/examples/read-and-write-data.txt:325]: did a global search in repo for `‘` and this was actually the only occurrence.

**Pull request merged**